### ISSUE TITLE
Allow switching between "Fixed key height" and "Fixed keyboard height" layout behavior

### ISSUE THREAD

**Issue by patrickgold** [florisboard/florisboard#2616]

**Comment by itsmartashub** [comment]
Hi there. I'm wonderng if this might include a fix for the small issue with the focused emoji tab? When the keyboard height is reduced, the emoji **tab indicator becomes less visible** and sometimes disappears.
It seems like there's not enough room in the emoji layout for the action key row, so it gets cut off (e.g. I'm the user of `85%` 🥲)
It becomes fully visible at `115%` of keyboard height.

This wasn’t an issue in the previous stable `v0.3.16`, which reading this desciption, makes it seem like you’re already assuming that.

- Florisboard `v0.4.0` both stable and [debug](https://github.com/florisboard/florisboard/actions/runs/11096588705) using default settings and theme, as well as with custom configs.
- `Poco F3` | `A13` | `HyperOS v1.0.6.0` | `Xiaomi.eu custom ROM`
- `OnePlus 5T` | `A14` | `RisingOS v5.2.1 Vanilla`
- Few more devices where it goes to fully visible from `125%`

![florisboard-emoji-tab-focus-cropped](https://github.com/user-attachments/assets/113d416a-4415-42f7-bb88-c279daf507a2)

**Comment by patrickgold** [comment]
The emoji panel is set to be rewritten anyways to accommodate for layout behavior customization and inline emoji search, then I will address that issue. However this in itself is completely separate.

Other question: how did you manage to get the same emoji multiple times into the history? That should be impossible...

**Comment by itsmartashub** [comment]
> The emoji panel is set to be rewritten anyways to accommodate for layout behavior customization and inline emoji search, then I will address that issue. However this in itself is completely separate.

Woah, nice! Thanks for the great news.


> Other question: how did you manage to get the same emoji multiple times into the history? That should be impossible...

Thank you, I was hoping you'd catch this from the screenshot as I wasn't sure if it was a bug or a feature. 
Just as I was about to write down about few minor things in the `rework/emoji-history` build, I realised it was already merged into the `main`. So I thought maybe these things aren't significant enough to bother you with a separate issue. 
Is it ok to add a comment to the already merged/closed PR as it seems more relevant than this issue, or?

**Comment by patrickgold** [comment]
Yes ou can just comment on the merged/closed PR that's perfectly fine!

**Comment by itsmartashub** [comment]
> Yes you can just comment on the merged/closed PR that's perfectly fine!
> 
> EDIT: I think #2617 should fix the emoji duplication you were able to reproduce

You're fast. Will test it soon
